test/internals/reply.test.js

```diff
@@ -177,6 +188,19 @@ test('within an instance', t => {
       })
     })
 
+    test('redirect to `/` - 5', t => {
+      t.plan(4)
+      sget({
+        method: 'GET',
+        url: 'http://localhost:' + fastify.server.address().port + '/redirect-onsend'
+      }, (err, response, body) => {
+        t.error(err)
+        t.strictEqual(response.statusCode, 200)
+        t.strictEqual(response.headers['content-type'], 'text/plain')
+        t.deepEqual(body.toString(), 'hello world!')
```

Comment:
Where this come from?

Reply:
? From the other tests that do the same thing.

Reply:
Oki, `sget` follows the redirection. Cound you use `inject` or `http` nodejs core module as the other tests?